packages/web-react/src/components/ModalComposed/useModalStyleProps.ts

```diff
@@ -0,0 +1,53 @@
+import classNames from 'classnames';
+import { useClassNamePrefix } from '../../hooks';
+
+export interface ModalStylesProps {
+  footerAlign?: 'left' | 'center' | 'right';
+  isExpandedOnMobile?: boolean;
+}
+
+export interface ModalStylesReturn {
+  /** className props */
+  classProps: {
+    root: string;
+    dialog: string;
+    title: string;
+    header: string;
+    body: string;
+    footer: string;
+    footerDescription: string;
+    footerActions: string;
+  };
+}
+
+export function useModalStyleProps({ footerAlign = 'right', isExpandedOnMobile }: ModalStylesProps): ModalStylesReturn {
+  const modalClass = useClassNamePrefix('Modal');
+  const modalComposedClass = `${modalClass}--composed`;
+  const modalExpandedOnMobileClass = `${modalClass}--expandOnMobile`;
+  const modalDialogClass = `${modalClass}__dialog`;
+  const modalTitleClass = `${modalClass}__title`;
+  const modalHeaderClass = `${modalClass}Header`;
+  const modalBodyClass = `${modalClass}Body`;
+  const modalFooterClass = `${modalClass}Footer`;
+  const modalFooterDescriptionClass = `${modalFooterClass}__description`;
+  const modalFooterActionsClass = `${modalFooterClass}__actions`;
+  const footerAlignClasses = {
+    left: `${modalFooterClass}--left`,
+    center: `${modalFooterClass}--center`,
+    right: `${modalFooterClass}--right`,
+  };
+  const classProps = {
+    root: classNames(modalClass, modalComposedClass, isExpandedOnMobile && modalExpandedOnMobileClass),
```

Comment:
https://www.npmjs.com/package/classnames#usage

```suggestion
    root: classNames(modalClass, modalComposedClass, { modalExpandedOnMobileClass: isExpandedOnMobile } ),
```

Reply:
fixed